src/util.rs

```diff
@@ -81,30 +73,62 @@ pub fn extract_str(bytes: &[u8]) -> Result<String, Utf8Error> {
 |----|----|----|----|----|----|
 | 13 | 14 | 15 | 16 | 17 | 18 |
  -----------------------------
+
+ Ajazz's AKP153(E) key index
+ -----------------------------
+| 0d | 0a | 07 | 04 | 01 | 10 |
+|----|----|----|----|----|----|
+| 0e | 0b | 08 | 05 | 02 | 11 |
+|----|----|----|----|----|----|
+| 0f | 0c | 09 | 06 | 03 | 12 |
+ -----------------------------
+
+ Ajazz's AKP815 key index
+  --------------
+ | 0f | 0e | 0d |
+ |----|----|----|
+ | 0c | 0b | 0a |
+ |----|----|----|
+ | 09 | 08 | 07 |
+ |----|----|----|
+ | 06 | 05 | 04 |
+ |----|----|----|
+ | 03 | 02 | 01 |
+  --------------
+
 */
 
 /// Converts Elgato key index to Ajazz key index
-pub fn elgato_to_ajazz(kind: &Kind, key: u8) -> u8 {
+pub fn elgato153_to_ajazz(kind: &Kind, key: u8) -> u8 {
```

Comment:
I think this function should be called `elgato_to_ajazz153`

Reply:
I didn't change the original naming. I only appended the model since not all models have the same key configuration.

One of those functions transforms from the elgato (linear) inage configuration to the ajazz image configuration and the other from button index. I can't review it right now since I'm on vacations and I'll come back in two weeks (I am writing this from memory)

Reply:
Yes. But there is no such thing as an Elgato 153. The function converts Elgato key indexes to Ajazz 153x ones, so it should be `elgato_to_ajazz153`.